src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -22,18 +34,26 @@ export const TodoFilter: React.FC = () => {
           type="text"
           className="input"
           placeholder="Search..."
+          value={query}
+          onChange={e => dispatch(
+            filterActions.setQuery(e.target.value as SortType),
```

Comment:
```suggestion
            filterActions.setQuery(e.target.value),
```
type casting is redundant here. setQuery should accept any string

Also, it's better to create a separate function for such handlers

Reply:
Argument of type 'string' is not assignable to parameter of type 'SortType'.ts(2345) поэтому я добавила уточнее типа